lib/apollo-federation/tracing/tracer.rb

```diff
@@ -35,15 +35,15 @@ module ApolloFederation
   module Tracing
     module Tracer
       # store string constants to avoid creating new strings for each call to .trace
-      EXECUTE_QUERY = 'execute_query'
+      EXECUTE_MULTIPLEX = 'execute_multiplex'
       EXECUTE_QUERY_LAZY = 'execute_query_lazy'
       EXECUTE_FIELD = 'execute_field'
       EXECUTE_FIELD_LAZY = 'execute_field_lazy'
 
       def self.trace(key, data, &block)
         case key
-        when EXECUTE_QUERY
-          execute_query(data, &block)
+        when EXECUTE_MULTIPLEX
+          execute_multiplex(data, &block)
```

Comment:
What is a multiplex? [The docs](https://graphql-ruby.org/api-doc/1.9.6/GraphQL/Tracing) are really sparse

Reply:
NVM [figured it out](https://graphql-ruby.org/queries/multiplex.html)